src/tricicl/metrics/normalized_accuracy.py

```diff
@@ -0,0 +1,31 @@
+from avalanche.evaluation.metrics import ExperienceAccuracy, StreamAccuracy
+
+"""
+Avalanche changed the behavior to the global training step 2w ago
+https://github.com/ContinualAI/avalanche/commit/89afc1cc620d88f09245920ac217a8be508e9e73#diff-6db76ef303657a86b96bc69cff62c8695bc4f046f683c43424ca8a2e608244f0
+which is not useful when plotting, getting the stream step is much more useful.
+"""
```

Comment:
I realized that part after freezing the version, so I made this patch instead of changing the version again